apps/glusterfs/app_config.go

```diff
@@ -23,13 +24,17 @@ type GlusterFSConfig struct {
 	Executor   string              `json:"executor"`
 	Allocator  string              `json:"allocator"`
 	SshConfig  sshexec.SshConfig   `json:"sshexec"`
+	CaiConfig  caiexec.CaiConfig   `json:"caiexec"`
 	KubeConfig kubeexec.KubeConfig `json:"kubeexec"`
 	Loglevel   string              `json:"loglevel"`
 
 	// advanced settings
 	BrickMaxSize int `json:"brick_max_size_gb"`
 	BrickMinSize int `json:"brick_min_size_gb"`
 	BrickMaxNum  int `json:"max_bricks_per_volume"`
+
+	// volume expand method
```

Comment:
Can u document what kind of expand methods we have?  Looks like the value is `quota` and default behavior is by brick, I think a complete solution should be `quota`, `brick` and default behavior is return error.